internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -4493,6 +4494,32 @@ func TestHandleGetMempoolInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleGetMiningInfo(t *testing.T) {
+	t.Parallel()
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleGetMiningInfo: ok",
+		handler: handleGetMiningInfo,
+		result: &types.GetMiningInfoResult{
+			Blocks:           432100,
+			CurrentBlockSize: 2782,
+			CurrentBlockTx:   7,
+			Difficulty:       2.8147398026656624e+10,
+			StakeDifficulty:  14428162590,
+		},
+	}, {
+		name:    "handleGetMiningInfo: handleGetNetworkHashPS error",
```

Comment:
I think the test names associated with errors could provide more detail if the they captured the context of the error generated. In this case a name like `"handleGetMiningInfo: invalid network hashes per sec"` would be a better fit, thoughts?

Reply:
Ok!